lib/travis/api/v3/renderer/owner.rb

```diff
@@ -19,5 +19,10 @@ def repositories
       repositories = query(:repositories).for_owner(@model)
       access_control.visible_repositories(repositories)
     end
+
+    def subscription
+      subscription = query(:subscription).for_owner(@model)
```

Comment:
Maybe we need to add some `access_control`  to verify the user has the rights to access the subscription ?

Reply:
I added it here: https://github.com/travis-ci/travis-api/pull/638/files#diff-701cdc602cd281c48cc828a3098bf074R65